tt_eager/tt_dnn/op_library/bmm/multi_core_reuse_mcast_2d_optimized/bmm_op_multi_core_reuse_mcast_2d_optimized.cpp

```diff
@@ -34,10 +34,10 @@ operation::ProgramWithCallbacks create_program_mcast_in0_in1(
     std::optional<UnaryWithParam> fused_activation,
     tt_metal::Buffer* in0_buffer, tt_metal::Buffer* in1_buffer, tt_metal::Buffer* bias_buffer, tt_metal::Buffer* out_buffer,
     tt::DataFormat in0_data_format, tt::DataFormat in1_data_format, tt::DataFormat bias_data_format, tt::DataFormat output_data_format,
-    bool in0_is_sharded, bool output_is_sharded,
+    bool in0_is_sharded, bool in1_is_sharded, bool output_is_sharded,
```

Comment:
Can we actually remove this from args that are passed in and just directly figure it out from buffers internally? It will be cleaner.

Reply:
make sense